### PR TITLE
Release notes update for 2.0

### PR DESCRIPTION
**Pull request by agupta49**
Release notes updated for 2.0.

### CODE REVIEW

File: docs/RELEASE-NOTES.rst [hunk 1]
```diff
@@ -4,24 +4,52 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+`````````````````````
++------------------+----------------------------------------------------------------+
+| CVE              | Comments                                                       |                                                                                               
++==================+================================================================+
+| CVE-2009-3555    | CIS disables renegotiation for all Custom ClientSSL            |
++------------------+----------------------------------------------------------------+
+
+Limitations
+```````````
+* CIS with agent cccl mode for OpenShift A/B route cannot be updated in BIGIP >=v14.1.x due to data group changes.
```
Comment (trinaths) [inline docs/RELEASE-NOTES.rst:52]: can be, 
 
CIS with `cccl` as agent, OpenShift A/B route cannot be updated in BIGIP >=v14.1.x due to data group changes.

File: docs/RELEASE-NOTES.rst [hunk 2]
```diff
@@ -4,24 +4,52 @@ Release Notes for BIG-IP Controller for Kubernetes
 Next Release
```
Comment (trinaths) [inline docs/RELEASE-NOTES.rst:4]: remove `Next Release`. We can have it once 2.0 is released.

Comment (agupta49) [reply docs/RELEASE-NOTES.rst]: Review comment incorporated.

File: docs/RELEASE-NOTES.rst (hunk 1 above)
Comment (agupta49) [reply docs/RELEASE-NOTES.rst]: Done.

**Pull request merged**